package.json

```diff
@@ -94,5 +94,8 @@
   "dependencies": {},
   "engines": {
     "node": ">=18"
+  },
+  "volta": {
+    "node": "18.0.0"
```

Comment:
Is this related to the PR somehow?

Reply:
Yes, this tells on which version of node tests will run.
Without it, setting `node-version-file: 'package.json'` will take version `20.2` based on the engine set to `"node": ">=18"`.
But I'd like to test the minimum node version we support and LTS instead.
See the improved comment on `.github/workflows/build-and-test.yml` below.

Reply:
removed this change.